subsys/modem/modem_cmux.c

```diff
@@ -830,6 +829,9 @@ static void modem_cmux_receive_handler(struct k_work *item)
 	/* Receive data from pipe */
 	ret = modem_pipe_receive(cmux->pipe, buf, sizeof(buf));
 	if (ret < 1) {
+		if (ret < 0) {
```

Comment:
something lesser than 1 must be lesser than 0

Reply:
Yes, so here this condition is equivalent to `ret != 0`. This is because a return value of 0 is not an error, thus we avoid logging an error in that case.

Reply:
> Yes, so here this condition is equivalent to `ret != 0`. This is because a return value of 0 is not an error, thus we avoid logging an error in that case.

ah sorry, my mind was reading `-1` instead of `1`